core/geometry/src/curve/internalContexts/geometry2d/ConstraintConstruction.ts

```diff
@@ -0,0 +1,271 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I renamed `ConstraintSet.ts` to `ConstraintConstruction.ts` to be consistent with `TangentConstruction.ts` 
However, I see the main API in this class i.e., `constructConstrainedCircles` and `ConstraintSet.constructConstrainedLines` are under `ConstraintSet` class so if you want to change the file name back to `ConstraintSet.ts` please go ahead.

Reply:
Done